pkg/private/pkg_files.bzl

```diff
@@ -61,9 +61,14 @@ _DestFile = provider(
     },
 )
 
-def _check_dest(content_map, dest, origin):
-    if dest in content_map:
-        fail("Duplicate output path: <%s>, declared in %s and %s" % (
+def _check_dest(content_map, dest, src, origin):
+    old_entry = content_map.get(dest)
+    # TODO(aiuto): This is insufficient but good enough for now. We should
+    # compare over all the attributes too. That will detect problems where
+    # people specify the owner in one place, but another overly broad glob
+    # brings in the file with a different owner.
```

Comment:
Please create and mention a ticket for tracking this issue.  This does seem good enough for now, but this could needlessly surprise users.

Perhaps also mention that you made this change in the commit's summary line.

Reply:
I folded that requirement into #385